lib/modules/dosomething/dosomething_signup/dosomething_signup.install

```diff
@@ -591,7 +591,7 @@ function dosomething_signup_update_7020() {
 /**
  * Adds campaign run nid column to the dosomething_signup table.
  */
-function dosomething_signup_update_7021() {
+function dosomething_signup_update_7023() {
```

Comment:
ah thanks for catching that! I got confused with the rebase. I'll change the number back and put between `dosomething_signup_update_7020` and `dosomething_signup_update_7022` so it's more readable.